packages/shylu/shylu_node/basker/src/shylubasker_def.hpp

```diff
@@ -634,26 +742,358 @@ namespace BaskerNS
     }
 */
 
+    /*{
```

Comment:
For a later cleanup follow up, we may want to add a macro that can be enabled to guard for this type output